src/classes/VOL_CTRL_PersonalSiteContactInfo_TEST.cls

```diff
@@ -30,61 +30,56 @@
 
 @isTest
 private with sharing class VOL_CTRL_PersonalSiteContactInfo_TEST {
-
+    private static VOL_Access_TEST.Stub accessMock = new VOL_Access_TEST.Stub();
+    
     //******************************************************************************************************
     // Test Code
 
+    private static void setAccessMock() {
+        VOL_CTRL_PersonalSiteContactInfo.access = (VOL_Access) Test.createStub(VOL_Access.class, accessMock);
+    }    
+    
+    private static testMethod void shouldCheckAccessOnSave(){
+        setAccessMock();
+		User guestUser = Util_UnitTest.guestUser;
```

Comment:
Indentation is off but do we even need to do anything here with the guestUser?

Reply:
I noticed this indentation too, but in my local code there is no indent. Either way, the line is removed now.